src/server/graphql/mutations/updateTask.js

```diff
@@ -59,7 +59,10 @@ export default {
     const {agendaId, content, status, assigneeId, sortOrder} = validUpdatedTask
     if (assigneeId) {
       const table = getIsSoftTeamMember(assigneeId) ? 'SoftTeamMember' : 'TeamMember'
-      const res = r.table(table).get(assigneeId)
+      const res = await r
```

Comment:
Is this is just "robusutification" to detect other areas where assigneeId was unset?

Reply:
nah, while digging around i just noticed that we weren't awaiting the promise, which meant that a nerd could set the assigneeId to "foo" and that'd cause a failed look up on Task assignee. we never saw it happen in prod because no one wanted to shoot themselves in the foot.

Reply:
cool cool 🏂